### PR TITLE
Update cluster5.cfg

### PR DESCRIPTION
**Pull request by Enryy**
I don't know if I wrote it correctly, but I would like to remove the radios from the tool shop... if it's not written correctly, tell me how to fix it

### CODE REVIEW

File: configs/cluster5.cfg [hunk 1]
```diff
@@ -29,13 +29,13 @@ extend_view_modes = false # adds all the native view modes
 excluded_jobs = "4,6"
 
 # replace or add new items into item stores
-store_overrides = "gun_store=pistol_ammo:42,sub_ammo:69;grocery_store=water:8;"
+store_overrides = "gun_store=pistol_ammo:42,sub_ammo:69;grocery_store=water:8;tool_shop=removeradio"
```
Comment (InZidiuZ) [inline configs/cluster5.cfg:32]: ...wut?

**Review by InZidiuZ — CHANGES_REQUESTED** [review]

File: configs/cluster5.cfg (hunk 1 above)
Comment (Enryy) [reply configs/cluster5.cfg:32]: As I wrote in the description, how can I remove an item from a shop? Is it possible to do this or can I just change the prices?

Comment (InZidiuZ) [reply configs/cluster5.cfg:32]: You can only change and add items.

Comment (Enryy) [reply configs/cluster5.cfg:32]: ah ok, could you give me an example of how to change an item

for example if I want to change pigeon's milk to radio

Comment (InZidiuZ) [reply configs/cluster5.cfg:32]: You can change an already-existing item and have a new price set. Not change it to another item.

**Review by InZidiuZ — APPROVED** [review]

**Pull request merged**